database/sql/src/main/scala/cromwell/database/slick/tables/MetadataEntryComponent.scala

```diff
@@ -62,10 +62,10 @@ trait MetadataEntryComponent {
   val metadataEntryIdsAutoInc = metadataEntries returning metadataEntries.map(_.metadataEntryId)
 
   val metadataEntriesForWorkflowExecutionUuid = Compiled(
-    (workflowExecutionUuid: Rep[String]) => for {
+    (workflowExecutionUuid: Rep[String]) => (for {
       metadataEntry <- metadataEntries
       if metadataEntry.workflowExecutionUuid === workflowExecutionUuid
-    } yield metadataEntry
+    } yield metadataEntry).sortBy(_.metadataTimestamp)
```

Comment:
Its certainly easier on the memory load for the cromwell server... but also feels like premature optimization to change from this if it works